test/test_post_an_answer.py

```diff
@@ -0,0 +1,14 @@
+from .test_base import *
+
+class Test_post_an_answer(TestBase):
+    def test_empty_post_answer(self):
+        self.app.post('/api/v1/questions', data = json.dumps({"title":"REST","description":"Introduction",
```

Comment:
The data you are sending is repeated in many places. I would appreciate it if you put it one place where you can access it